redlines/utils/markdown_processor.py

```
@@ -0,0 +1,166 @@
+    # ------------------------------------------------------------------
+    # Internal helpers
+    # ------------------------------------------------------------------
+    async def _docling_to_markdown(self, payload: bytes, name: str) -> Optional[str]:
+        if not self.enable_docling:
+            return None
+
+        try:
+            converter = DocumentConverter()
+            stream = DocumentStream(name=name, stream=BytesIO(payload))
+            result = await asyncio.to_thread(converter.convert, stream)
+        except Exception as exc:  # pragma: no cover - docling heavy path
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)